docs/source/troubleshooting/technicalreport.rst

```diff
@@ -0,0 +1,23 @@
+========================
+Technical Support Report
+========================
+
+Hardware Configuration Utility can generate a Technical Support Report for a NI Linux Real Time system which contains information about the system that is helpful in troubleshooting issues.
```

Comment:
Why is this documentation in the NILRT docs, and not in HWCU? I would expect the only thing we would document is how to provoke this workflow from the shell *without* HWCU, if that is even possible.

Reply:
Technical Report is platform specific. Reports could be generated for windows host, Linux RT target (and presumably pharlap and Linux hosts as well).
Since existing HWCU documentation doesn't go into details about the contents of the technical report for any platform, I assume this is the level at which they want to keep HWCU's documentation.

We however would like to document NI Linux RT specific technical report and nilrt-docs seemed like the best place.

Reply:
>Since existing HWCU documentation doesn't go into details about the contents of the technical report

Neither do the NILRT docs. :)

If there are parts of this workflow that are owned by the system, then I think those are fine to document here. But if the content and use of the technical report is defined by HWCU (as I think our prior conversations have me believing), then they are the product that should own, test, and document it.

I wouldn't go to my distro's documentation to learn how and where Steam saves its crash logs.

Reply:
The info I added here regarding adding more files/dirs to the report is not owned by RTOS.. but if you think nilrt-docs is not the place to put it, I can remove it; but I'm not going to go through the trouble of having it be documented in ni.com.